# Ball position and speed
Ball ball

# Paddle position and speed
Paddle paddle1
Paddle paddle2

# Player scores
Game game
